RELEASE.md

```diff
@@ -6,7 +6,16 @@ The Kubernetes Windows GMSA project is released on an as-needed basis. The proce
 1. All [OWNERS](OWNERS) must LGTM this release issue
 1. An OWNER runs `git tag -s $VERSION` from `master` branch and pushes the tag with `git push $VERSION`
 1. An OWNER promotes the `gcr.io/k8s-staging-gmsa-webhook/k8s-gmsa-webhook` image built the tagged commit.
-    1. Insuctions TBD
+    1. Follow setup steps for `kpromo` from [here](https://github.com/kubernetes-sigs/promo-tools/blob/main/docs/promotion-pull-requests.md#preparing-environment) if needed
+    1. Tag the desired container image in the [staging registry](https://console.cloud.google.com/gcr/images/k8s-staging-gmsa-webhook/GLOBAL) as `$VERSION`
+    1. Run `kpromo pr` to open a pull request to have tagged container image promted from staging to release registries
+
+        ```bash
+        kpromo pr --project gmsa-webhook --tag $VERSION --reviewers "@jayunit100 @jsturtevant @marosset" --fork {your github fork}
```

Comment:
```suggestion
        kpromo pr --project gmsa-webhook --tag $VERSION --reviewers "@jayunit100 @jsturtevant @marosset" --fork {your github fork of kubernetes/k8s.io}
```

Reply:
I think?

Reply:
Just your repo ie `marosset` or `jsturtevant`.
I agree this is confusing but that is how the docs/tool help refer to this...

Reply:
Well, i think the docs call it USER_FORK

Reply:
I'll say {your github username}.
I see that in some places.